internal/pkg/template/templates/workloads/partials/cf/nlb.yml

```diff
@@ -11,57 +11,58 @@ PublicNetworkLoadBalancer:
             !Sub '${AppName}-${EnvName}-PublicSubnets'
     Type: network
 
-NLBListener:
+{{- range $listener := .NLB.Listener}}
+NLBListenerFor{{ $listener.Port }}:
```

Comment:
Is this backwards compatible?

If someone has already deployed an NLB, by changing the LogicalID it will result in the listener to be deleted and then recreated. My guess is that it will lead to downtime 💭 

:thinkspin: methinks that we would need something like:

```
# Keep existing resource
NLBListener: 
  ...

{{- if gt (len .NLB.Listener) 1 }}
{{- range $i, $listener := .NLB.Listener }}
NLBListener{{$i + 1}}:
   # Remaining properties...
{{- end }}  
```

Reply:
That's a great call out. I am not sure if this will cause a downtown but definitely block customers with the error saying Listener for particular port already exist. I will make the required changes.